test/e2e/storage/csi_mock_volume.go

```diff
@@ -1973,6 +1974,113 @@ var _ = utils.SIGDescribe("CSI mock volume", func() {
 			})
 		}
 	})
+
+	ginkgo.Context("CSI DeleteVolum", func() {
+		trackedCalls := []string{
+			"DeleteVolume",
+		}
+
+		tests := []struct {
+			name          string
+			expectedCalls []csiCall
+
+			// Called for each DeleteVolume calls, with counter incremented atomically before
+			// the invocation (i.e. first value wool be 1).
+			deleteVolumeHook func(counter int64) error
+		}{
+			{
+				name: "CSI: delete pv prior",
+				expectedCalls: []csiCall{
+					{expectedMethod: "DeleteVolume", expectedError: codes.OK},
+				},
+			},
+		}
+
+		for _, t := range tests {
+			test := t
+			ginkgo.It(test.name, func() {
+				e2eskipper.SkipUnlessFeatureGateEnabled(kubefeatures.HonorPVReclaimPolicy)
```

Comment:
Careful! A test cannot use feature gate checks. You have to add `Feature: HonorPVReclaimPolicy` to the test name.

Reply:
only add testcase to in-tree plugins, I had deleted it.